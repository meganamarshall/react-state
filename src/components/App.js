import React from 'react';
import ColorPicker from './ColorPicker';

export default function App() {
  const colors = ['red', 'green', 'blue'];
  return <ColorPicker colors={colors} />;
}
