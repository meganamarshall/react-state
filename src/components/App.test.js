import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App component', () => {
  it('renders the app', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
