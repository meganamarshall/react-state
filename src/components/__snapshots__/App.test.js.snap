// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app component renders the app 1`] = `<RandomColor />`;
